### PR TITLE
Articles handling branch

### PR DESCRIPTION
**Pull request by matdon90**
Implemented functionalities with tests:

- User can add article.
- User can edit article.
- User can delete article.
- Anyone can see articles list.
- Anyone can see selected article details.

Roles authorization needed to provide only function that only admin can add, edit or delete article..

### CODE REVIEW

File: SK.Application.IntegrationTests/Articles/Commands/CreateArticleTest.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        }
+
+        [Test]
+        public void ShouldRequireContent()
+        {
+            //arrange
+            var command = new CreateArticleCommand()
+            {
+                Id = Guid.NewGuid(),
+                Title = "Article Title",
```
Comment (UgzSourceCode) [inline SK.Application.IntegrationTests/Articles/Commands/CreateArticleTest.cs:91]: you haven't to write real text for unit tests, try use faker.net or anything else ;)

File: SK.Application.IntegrationTests/Articles/Commands/CreateArticleTest.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+{
+    using static Testing;
+
+    public class CreateArticleTest : TestBase
+    {
+        [Test]
+        public async Task ShouldCreateArticle()
+        {
+            //arrange
+            var loggedUser = await RunAsUserAsync("scott101@localhost", "Pa$$w0rd!");
```
[the remark this answers is not in the document]
Comment (matdon90) [reply SK.Application.IntegrationTests/Articles/Commands/CreateArticleTest.cs:22]: All similar tests from Events and Articles combined together into TestCases.

File: SK.Application.IntegrationTests/Articles/Commands/CreateArticleTest.cs (hunk 1 above)
Comment (matdon90) [reply SK.Application.IntegrationTests/Articles/Commands/CreateArticleTest.cs]: Used Bogus for .NET to fake data in Articles and Events tests.

File: SK.Application.IntegrationTests/Events/Commands/EditEventTest.cs [hunk 3]
```diff
@@ -56,172 +54,69 @@ public async Task ShouldUpdateTestValue()
...
-                City = "Internet",
-                Venue = "Discord"
-            };
-
-            //act
-
-            //assert
-            FluentActions.Invoking(() =>
-                SendAsync(command)).Should().Throw<ValidationException>();
+                yield return new TestCaseData(null, new Faker("en").Lorem.Sentence(), new Faker("en").Lorem.Sentence(5), new Faker("en").Lorem.Word(), new Faker("en").Lorem.Word(), new Faker("en").Lorem.Sentence(1)).SetName("EventDateMissingTest");
```
[the remark this answers is not in the document]
Comment (matdon90) [reply SK.Application.IntegrationTests/Events/Commands/EditEventTest.cs:63]: Done

**Pull request merged**